packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -127,6 +128,7 @@ describe("Compile task", () => {
 	it("should return correct ASCCache from CompileOp", async () => {
```

Comment:
Could you add test suit follow this suit? 
1. compile and ASACache to cache file. 
2. Load this file and verify we stored scParams.